pkg/daemon.py

```diff
@@ -35,19 +37,21 @@ async def daemon(
     """
     parameters = {"version": version}
     config_file_name=test_name+".yml"
-    argDict = {
+    option_arguments = {
         "config": config_file_name,
         "output_path": "output.csv",
         "hunter_analyze": True,
-        "output_format": "json",
+        "anomaly_detection": False,
```

Comment:
While we already have `hunter_analyze` to indicate that this is related to change point detection, do we need `anamaly_detection` as an argument here? Vice versa applies to the endpoint `/daemon/anomaly` too.